### PR TITLE
Feat : Add Home Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Home Page
I have started  "Trendy-T" an E-Commerce website. In which I have added a home page which contains several features.
1. Home Screen
2. Category
3. Offers
4. Product
5. Service

### CODE REVIEW

File: Home-Page/responsive.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  .card-product-category {
+    margin: 0 1rem;
+  }
+  .card-offer__container,
+  .card-product,
+  .card__container--demo {
+    margin: 8px 1rem;
+  }
+}
+@media screen and (max-width: 500px) {
```
Comment (Raghav888) [inline Home-Page/responsive.css:26]: in media query use em

**Review by Raghav888 — APPROVED** [review]

File: Home-Page/responsive.css (hunk 1 above)
Comment (SaurabhChawda) [reply Home-Page/responsive.css:26]: Okay , ThankYou

File: Home-Page/responsive.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    margin: 1rem 2em;
+  }
+}
+@media screen and (max-width: 750px) {
+  .card-product-category {
+    grid-template-columns: 1fr;
+  }
+  .card__container--demo {
+    grid-template-columns: 1fr;
+  }
```
Comment (mariyasada) [inline Home-Page/responsive.css]: you can combine this class

**Review by mariyasada — APPROVED** [review]

File: Home-Page/responsive.css (hunk 2 above)
Comment (SaurabhChawda) [reply Home-Page/responsive.css]: okay , ThankYou

**Pull request merged**